core/src/actions/appRefreshQuery.ts

```diff
@@ -0,0 +1,205 @@
+import { AuthenticatedAction } from "../classes/actions/authenticatedAction";
+import { AppRefreshQuery } from "../models/AppRefreshQuery";
+import { ConfigWriter } from "../modules/configWriter";
+import { APIData } from "../modules/apiData";
+import { AppRefreshQueryOps } from "../modules/ops/appRefreshQuery";
+import { CLS } from "../modules/cls";
+
+export class AppRefreshQueriesList extends AuthenticatedAction {
+  constructor() {
+    super();
+    this.name = "appRefreshQueries:list";
```

Comment:
I don't think there is a list action in the UI in use here?  The AppRefreshQueries are linked to from the app, and there can be only one